app/controllers/assessments_controller.rb

```diff
@@ -159,8 +159,12 @@ def find_contact_id; end
 
   def log_assessment
     @need = Need.new(assessment_params.merge(contact_id: @contact.id, start_on: Date.today))
+    @need.name = "#{@need.category} call - #{Time.zone.now.strftime('%d/%m/%y')}"
```

Comment:
Should this follow the same date formatting that we use for contacts? `%d/%m/%Y`?